src/entity-manager/EntityManager.ts

```diff
@@ -1291,7 +1291,7 @@ export class EntityManager {
     ): Repository<Entity> {
         // find already created repository instance and return it if found
         const repository = this.repositories.find(
-            (repository) => repository.target === target,
+            (repo) => repo.target === target,
```

Comment:
redundant change. please remove it.

Reply:
There is a variable on 1293 line with the samename and and this was breaking the test